.lintstagedrc.json

```diff
@@ -2,5 +2,9 @@
   "*.js": [
     "eslint --fix",
     "git add"
+  ],
```

Comment:
This part is not needed.

`remark` may help to address this kind of tests but it is not yet installed.

I think you can omit this addition.

Reply:
@antleblanc as seen together this is to lint `js` inside `md` files 
I agree we should uniformize our linters, adding `remark` could be addressed as another PR